sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/service/schema/TableManagerTest.java

```diff
@@ -337,15 +340,9 @@ public void created() {
           .when(bridgeService)
           .executeQuery(any(), any());
 
-      UniAssertSubscriber<Boolean> result =
-          tableManager
-              .ensureValidDocumentTable(namespace, collection)
-              .subscribe()
-              .withSubscriber(UniAssertSubscriber.create());
+      tableManager.ensureValidDocumentTable(namespace, collection).await().indefinitely();
```

Comment:
this assertion is not good, should be:

```
      Schema.CqlTable result =
          tableManager
              .ensureValidDocumentTable(namespace, collection)
              .subscribe()
              .withSubscriber(UniAssertSubscriber.create())
              .awaitItem()
              .assertCompleted()
              .getItem();

      assertThat(result).isNotNull();
```      

In order for test to pass, you need to define two answers for the `describeKeyspace` mock first without the table, then with one:

```
      Schema.CqlKeyspaceDescribe keyspace =
          Schema.CqlKeyspaceDescribe.newBuilder()
              .setCqlKeyspace(Schema.CqlKeyspace.newBuilder().setName(namespace))
              .build();

      Schema.CqlKeyspaceDescribe validTableAndKeyspace =
          getValidTableAndKeyspace(namespace, collection);

      doAnswer(
              invocationOnMock -> {
                StreamObserver<Schema.CqlKeyspaceDescribe> observer =
                    invocationOnMock.getArgument(1);
                observer.onNext(keyspace);
                observer.onCompleted();
                return null;
              })
          .doAnswer(
              invocationOnMock -> {
                StreamObserver<Schema.CqlKeyspaceDescribe> observer =
                    invocationOnMock.getArgument(1);
                observer.onNext(validTableAndKeyspace);
                observer.onCompleted();
                return null;
              })
          .when(bridgeService)
          .describeKeyspace(any(), any());
```